web/src/user_groups.ts

```diff
@@ -423,7 +423,7 @@ export function check_group_can_be_subgroup(
 
 export function get_potential_subgroups(target_user_group_id: number): UserGroup[] {
     const target_user_group = get_user_group_from_id(target_user_group_id);
-    return get_all_realm_user_groups().filter((user_group) =>
+    return get_all_realm_user_groups(true).filter((user_group) =>
```

Comment:
Why this is being passed as true? We do not want deactivated groups to be allowed as subgroups. I know we exclude them in `check_group_can_be_subgroup` but still we can exclude them at the start.